src/js/services/isAdItem.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
maybe refactor this (later) to separate npm module since it's also used in media-hls